e2e-tests/nextjs/sentry.client.config.ts

```diff
@@ -31,9 +31,7 @@ Sentry.init({
   ],
 });
 
-if (process.env.NODE_ENV === 'development') {
-  Spotlight.init({
-    debug: true,
-    integrations: [customIntegration()],
-  });
-}
+Spotlight.init({
```

Comment:
I'm aware that this is just an e2e test app but is this how we want users to use Spotlight in the future?
I'm probably missing something but if this guard is removed, spotlight overlay code ends up in the production build of the NextJS application. Currently we document spotlight setup with this guard in place so that users don't accidentally include it in their prod bundles. 

The reason is that bundlers are smart enough to remove code within the `process.env.NODE_ENV` check if the condition evaluates to false at build time.